views/courses/edit-course.hbs

```diff
@@ -93,9 +93,9 @@
 
 				<div class="form-group">
 					<label for="input0">{{$t "courses.global.label.courseColor"}}</label>
-					<div class="color-picker">
+					<div class="color-picker" data-testid="color-picker">
 						{{#if course.color}}
-							<div class="color-picker__item">
+							<div class="color-picker__item" data-testid="color-picker">
```

Comment:
```suggestion
							<div class="color-picker__item" data-testid="item">
```
I think that data-testid should be `item` to be consistent with the create-course case: https://github.com/hpi-schul-cloud/schulcloud-client/blob/main/views/courses/create-course.hbs#L69